src/LobbyForm.tsx

```diff
@@ -56,16 +75,27 @@ function Join({ socket }: HostProps): ReactElement {
   return (
     <div className={css(hostStyles.row)}>
       <div>
-        <input
-          className={css(hostStyles.input)}
-          placeholder="Join a lobby using a name"
+        <TextInput
+          width={223}
+          placeholder="Join a lobby"
+          theme="purple"
           onChange={onChangeLobbyID}
           value={lobbyID}
         />
       </div>
       <div>
-        <button onClick={onJoinLobby} className={css(hostStyles.button)}>
-          &gt;
+        <button
+          onClick={onJoinLobby}
+          className={css(hostStyles.button, hostStyles.button2)}
+        >

```

Comment:
why do we need this button? shouldn't the Button component replace this completely?